src/main/java/org/jenkinsci/test/acceptance/plugins/dashboard_view/BuildStatisticsPortlet.java

```diff
@@ -0,0 +1,54 @@
+package org.jenkinsci.test.acceptance.plugins.dashboard_view;
+
+import org.jenkinsci.test.acceptance.po.Describable;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+/**
+ * The basic Build statistics portlet shipped with the dashboard view plugin.
+ *
+ * @author Rene Zarwel
+ */
+@Describable("Build statistics")
+public class BuildStatisticsPortlet extends AbstractDashboardViewPortlet {
+
+  public enum Jobtype{
+    FAILED(2), UNSTABLE(3), SUCCESS(4), PENDING(5),
+    DISABLED(6), ABORTED(7), NOT_BUILT(8), TOTAL(9);
+
+    private final int row;
+    Jobtype(int r){
+      row =r;
+    }
+  }
+
+  public BuildStatisticsPortlet(DashboardView parent, String path) {
+    super(parent, path);
+  }
+
+  /**
+   * Get the Buildstatistics table as {@link WebElement}
+   * @return table
+   */
+  public WebElement getTable(){
```

Comment:
I just use the public `find` method here, so i don't see a reason to block the accessibility.